src/lib/error-rate-check.js

```diff
@@ -0,0 +1,23 @@
+const nHealth = require('n-health');
+
+module.exports = (appName) => (
+	nHealth.runCheck({
+		type: 'graphiteThreshold',
+		metric: `
+			divideSeries(
+				sumSeries(
+					next.heroku.${appName}.web_*_${process.env.REGION || '*'}.express.default_route_GET.res.status.500.count
+				),
+				sumSeries(
```

Comment:
more like this: https://github.com/Financial-Times/next-myft-page/blob/master/server/health-checks/50x.js ?